### PR TITLE
refactor filter component 

### PR DESCRIPTION
**Pull request by ciremusyoka**
Addressing issue #134

### CODE REVIEW

File: src/lib/components/Filter/Filter.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {
   buildFilterState,
   clearFilterState,
   lngLat,
+  mergeFilters,
+  buildFiltersMap
 } from 'gisida';
```
Comment (cKellyDesign) [inline src/lib/components/Filter/Filter.js]: @ciremusyoka What version / branch of gisida does this branch required?

**Review by cKellyDesign — COMMENTED** [review]
Nice @ciremusyoka! What's the status of this PR? #134 seems closed, if we get the tests passing is this ready to merge to staging?

File: src/lib/components/Filter/Filter.js (hunk 1 above)
Comment (ciremusyoka) [reply src/lib/components/Filter/Filter.js]: @cKellyDesign gisida branch is map-refactor which have also created a pull request (https://github.com/onaio/gisida/pull/387)  for.

Comment (cKellyDesign) [reply src/lib/components/Filter/Filter.js]: @ciremusyoka looks like the Gisida branch was merged, should we merge this now as well?

**Pull request closed without merging**